tests/unit/src/unit/issues/Issue6278.hx

```diff
@@ -0,0 +1,16 @@
+package unit.issues;
```

Comment:
Hm, probably. Not sure where exactly it would go though - there's a `TestFileInput` that deals with `sys.io.File`, but that doesn't really fit.